Models/biophysics_calibration/pyabc_cellmodeller.py

```diff
@@ -28,12 +28,20 @@
 
 # Local/custom modules
 sys.path.append(os.path.join(os.path.expanduser('~'), 'CellModeller-ingallslab/Scripts/summaryStatistics/biophysics/'))
-import anisotropy
-import aspect_ratio
-import density_calculation
-import growth_rate_analysis
+sys.path.append(os.path.join(os.path.expanduser('~'), 'CellModeller-ingallslab/Scripts/RunSS_henry/scripts/SummaryStatistic'))
```

Comment:
What modules are being imported from RunSS_henry? It seems like the summary statistics are imported from run_ss_on_exp_sim.

Reply:
We can just delete that line, nothing is depended on it